pkg/controller/kabaneroplatform/landing.go

```diff
@@ -98,6 +104,73 @@ func deployLandingPage(k *kabanerov1alpha1.Kabanero, c client.Client) error {
 	return err
 }
 
+func cleanupLandingPage(k *kabanerov1alpha1.Kabanero, c client.Client) error {
+	err := removeWebConsoleCustomization(k)
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			return err
+		}
+	}
+
+	clientset, err := getClient()
+	deploymentClient := clientset.AppsV1().Deployments(k.GetNamespace())
+	if err != nil {
+		return err
+	}
+
+	deletePolicy := metav1.DeletePropagationForeground
+	err = deploymentClient.Delete("kabanero-landing", &metav1.DeleteOptions{
+		PropagationPolicy: &deletePolicy})
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			return err
+		}
+	}
+	rev, err := resolveSoftwareRevision(k, "landing", k.Spec.AppsodyOperator.Version)
```

Comment:
On this line, and also line 30, I don't understand why we're reading the AppsodyOperator version.

Reply:
In the latest commit I changed both places to be Landing version.